test/module/shared_model/command_mocks.hpp

```diff
@@ -36,44 +36,59 @@ namespace shared_model {
       MOCK_CONST_METHOD0(get, const Command::CommandVariantType &());
     };
 
+    struct SpecificMockCommandBase {};
+
+    template <typename T>
+    struct SpecificMockCommand : public T, public SpecificMockCommandBase {
+      using SpecificCommand = T;
+    };
+
     struct MockAddAssetQuantity
-        : public shared_model::interface::AddAssetQuantity {
+        : public SpecificMockCommand<
+              shared_model::interface::AddAssetQuantity> {
```

Comment:
You are already in the namespace
```suggestion
              AddAssetQuantity> {
```

Reply:
Please apply the same fix for the rest commands.